### PR TITLE
Timestamp align

### LINKED ISSUE #1228: inconsistent spacing of timestamp
### Steps to reproduce

The bubble layout is great. However, I've seen some inconsistencies with timestamp spacing, as indicated in the screenshot. Messages are numbered from the first one that isn't an image:
Messages 1, 2 and 3 are fine.
Messages 4 and 6 have a bit less than an extra line without any necessity. Message 5 needs the extra line for the timestamp, but it seems to be adding a full height line there despite the timestamp being smaller than regular text.
I can't really tell what triggers awkward spacing, but I suppose it's some kind of formatting, such as markdown or just a reply being rendered.

Screenshot_20230905-200128

### Your phone model

_No response_

### Operating system version

android 13

### Application version and app store

Element X 0.1.5

### Homeserver

_No response_

### Will you send logs?

No

### Are you willing to provide a PR?

No

### PR DESCRIPTION
**Pull request by bmarty**
## Type of change

- [ ] Feature
- [x] Bugfix
- [ ] Technical
- [ ] Other :

## Content

Add a Spacer along with the HtmlDocument composable.

The computation is quite empiric, end there is still a case which is not working as expected (see in the screenshot).

## Motivation and context

Closes #1228

## Screenshots / GIFs

See the recorded sc.

## Tests

- Step 1
- Step 2
- Step ...

## Tested devices

- [ ] Physical
- [ ] Emulator
- OS version(s):

## Checklist

- [ ] Changes have been tested on an Android device or Android emulator with API 23
- [ ] UI change has been tested on both light and dark themes
- [ ] Accessibility has been taken into account. See https://github.com/vector-im/element-x-android/blob/develop/CONTRIBUTING.md#accessibility
- [ ] Pull request is based on the develop branch
- [ ] Pull request includes a new file under ./changelog.d. See https://github.com/vector-im/element-x-android/blob/develop/CONTRIBUTING.md#changelog
- [ ] Pull request includes screenshots or videos if containing UI changes
- [ ] Pull request includes a [sign off](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#sign-off)
- [ ] You've made a self review of your PR

### CODE REVIEW

File: tests/uitests/src/test/snapshots/images/ui_S_t[f.messages.impl.timeline.components_null_TimelineItemEventRowTimestampDark_0_null_0,NEXUS_5,1.0,en].png
Comment (bmarty) [inline tests/uitests/src/test/snapshots/images/ui_S_t[f.messages.impl.timeline.components_null_TimelineItemEventRowTimestampDark_0_null_0,NEXUS_5,1.0,en].png:1]: There is a case which is not OK here :/

File: tests/uitests/src/test/snapshots/images/ui_S_t[f.messages.impl.timeline.components_null_TimelineItemEventRowTimestampDark_0_null_3,NEXUS_5,1.0,en].png
Comment (jmartinesp) [inline tests/uitests/src/test/snapshots/images/ui_S_t[f.messages.impl.timeline.components_null_TimelineItemEventRowTimestampDark_0_null_3,NEXUS_5,1.0,en].png:1]: It seems like something went wrong in the 4th message.

**Review by jmartinesp — APPROVED** [review]
LGTM, thanks!

File: tests/uitests/src/test/snapshots/images/ui_S_t[f.messages.impl.timeline.components_null_TimelineItemEventRowTimestampDark_0_null_1,NEXUS_5,1.0,en].png
Comment (bmarty) [inline tests/uitests/src/test/snapshots/images/ui_S_t[f.messages.impl.timeline.components_null_TimelineItemEventRowTimestampDark_0_null_1,NEXUS_5,1.0,en].png:1]: On the last case, the space is too big. In fact this PR only improve the Document behavior when there is only one line of text.

**Pull request merged**